src/entities/committer.ts

```diff
@@ -36,7 +36,7 @@ export const defaultCommitter = {
 export default class Committer {
 	_contract?: PoolCommitter;
 	address: string;
-	provider: ethers.providers.JsonRpcProvider;
+	provider: ethers.providers.JsonRpcProvider | ethers.providers.JsonRpcSigner;
```

Comment:
oh something I came across yesterday was actually to broaden this outside of `JsonRpcProvider` because I'm using a websocket provider on the api and had to cast it to the right type. would be nice if it just accepted it

Reply:
Opened it up to all providers